src/Modules/Reputation/interfaces/ReputationTokenInterface.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface ReputationTokenInterface {
+    struct ReputationAccountInfo {
+        uint256 locked;
+        uint256 lastDecayEpoch;
+        uint256 frozenUntilEpoch;
+    }
+
+    event ReputationFrozen (
+        address indexed account,
+        uint256 frozenUntilEpoch
+    );
+
+    event ReputationLocked (
```

Comment:
Right now, the events are written as if they are state-focused however Ethereum events are action-focused. This means, that instead of having a different event for Minting, Burning and Transferring, we just have one Transfer (action) event that represents different states of those actions.

Labor markets should adopt the same type of events. There is a lot of potential information that we can cover and should cover in the coming days to confirm that as a team we are fully aligned on when and why we use events and the rules associated with doing so.

Right now, please just get these to be action-based and then we can worry about data-types in a subsequent PR following the discovery of event interaction at the event consumption layer.